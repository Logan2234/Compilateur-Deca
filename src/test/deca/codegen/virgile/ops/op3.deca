// Created by Virgile, 10/01/2022
// test operations.
{
    int a = 3 * 4 + 2 % 3;
    println(a);
}