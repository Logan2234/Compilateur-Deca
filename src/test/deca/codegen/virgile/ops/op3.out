  IMA -- Fichier ./src/test/deca/codegen/virgile/ops/op3.ass introuvable
